examples/UVC/uvc_disparity.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import time
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-fb', '--flash-bootloader', default=False, action="store_true")
+parser.add_argument('-f',  '--flash-app',        default=False, action="store_true")
+parser.add_argument('-l',  '--load-and-exit',    default=False, action="store_true")
+args = parser.parse_args()
```

Comment:
Maybe remove this flashing capability for disparity/mono - it only makes sense for uvc_rgb, more as a showcase

Reply:
Uvc node is not yet added since not all tests were run yet. The .rsts should have references to it though.